src/test/software/amazon/qldb/QldbSessionTest.java

```diff
@@ -48,13 +48,7 @@
 import software.amazon.awssdk.core.exception.SdkClientException;
 import software.amazon.awssdk.core.exception.SdkException;
 import software.amazon.awssdk.core.exception.SdkServiceException;
-import software.amazon.awssdk.services.qldbsession.model.AbortTransactionRequest;
-import software.amazon.awssdk.services.qldbsession.model.BadRequestException;
-import software.amazon.awssdk.services.qldbsession.model.InvalidSessionException;
-import software.amazon.awssdk.services.qldbsession.model.OccConflictException;
-import software.amazon.awssdk.services.qldbsession.model.QldbSessionException;
-import software.amazon.awssdk.services.qldbsession.model.RateExceededException;
-import software.amazon.awssdk.services.qldbsession.model.SendCommandRequest;
+import software.amazon.awssdk.services.qldbsession.model.*;
```

Comment:
Don't we prefer to explicitly state all the dependencies instead of `*`?

Reply:
Updated to use explicit imports